### ISSUE TITLE
Have a way to regen bakeddata for a single key or a key prefix

### ISSUE THREAD

**Issue by Manishearth** [unicode-org/icu4x#4227]
`cargo make bakeddata <component>` is slow for many components (especially datetime), but usually you know which keys you've actually changed. Would be nice if you could run `cargo make bakeddata datetime datetime/symbols/` to generate all symbols keys. 

Triage suggestion: some form of backlog, very much a "nice to have internal improvement"

**Comment by robertbastian** [comment]
This will be annoying to do, because `make-bakeddata` (and baked exporter) works on a data crate at a time.

**Comment by Manishearth** [comment]
Yeah I expect so, this is an extremely aspirational issue. It would involve some refactoring.
